settings.gradle.kts

```diff
@@ -34,6 +34,7 @@ include(":ui:common")
 include(":ui:root")
 include(":ui:signed-in")
 include(":ui:signed-out")
+include(":ui:desktop-webview")
```

Comment:
@crocsandcoffee I already decalre the `:ui:desktop-webview ` module here. Do we need to add it somewhere else?

Reply:
Oh weird. I didn't see this file in the PR diff. No this should be it. Thanks! 👍